justfile

```diff
@@ -118,18 +126,15 @@ protoc:
     rm protoc.zip
   fi
 
-
-# Installs python dependencies for testing
-venv:
+@venv:
   python3 -c "import virtualenv" || python3 -m pip --quiet install virtualenv
   python3 -m virtualenv {{VENV}} --quiet
-  {{VENV_BIN}}/python -m pip install poetry
-  {{VENV_BIN}}/poetry -C tests install
-
+  
 
-# Runs pytest in the tests directory.
-pytest *args: 
-	{{VENV_BIN}}/poetry -C tests run pytest --rootdir={{invocation_directory()}}/tests {{ if args == "" {'tests'} else {args} }}
+## Set up virtual environment and install requirements
+@requirements: venv
+  {{VENV_BIN}}/python -m pip --quiet install --upgrade pip
```

Comment:
I think this checks pypi for an up-to-date pip every time, including on every `fmt` call, right?